__tests__/jesttest.tsx

```diff
@@ -92,6 +94,43 @@ describe("LibraryContainer", function () {
     expect(libraryItem.prop('data').childItems[0].childItems[0].text).to.equal("Child1");
     expect(libraryItem.prop('data').childItems[0].childItems[1].text).to.equal("Child2");
   });
+   describe("Test mouse click event and change in expand state", function () {
```

Comment:
@monikaprabhu Please help to extract this `describe()` block to be on the same level with the other `describe()` blocks in the same file, as we have clarified that we have no intention to nest the blocks.

I will add my new `describe()` and `it()` blocks accordingly after your change.

Thanks :wink: